happytransformer/adaptors/berts.py

```diff
@@ -0,0 +1,52 @@
+from .adaptor import Adaptor
+from transformers import (
+    BertForMaskedLM, BertTokenizerFast,
+    BertForNextSentencePrediction, BertForQuestionAnswering,
+    BertForSequenceClassification,
+
+    DistilBertForMaskedLM, DistilBertTokenizerFast,
+    DistilBertForSequenceClassification, DistilBertForQuestionAnswering,
+
+    RobertaForMaskedLM, RobertaTokenizerFast,
+    RobertaForQuestionAnswering, RobertaForSequenceClassification,
+
+    AlbertForMaskedLM, AlbertTokenizerFast, AlbertForQuestionAnswering,
+    AlbertForSequenceClassification
+)
+
+class BertAdaptor(Adaptor):
```

Comment:
We don't have to do this now, but she should be in alphabetical order

Reply:
I think BERT should be at the top because it's a simpler case. It would seem odd to start with a model that has an exception